Tests/XCUITests/DefaultBrowserTests.swift

```diff
@@ -2,37 +2,12 @@
 // License, v. 2.0. If a copy of the MPL was not distributed with this
 // file, You can obtain one at http://mozilla.org/MPL/2.0/
 
-import XCTest
+import Foundation
```

Comment:
Nit: is this on purpose? I think we could remove this test suite... the test is disabled and with the new way it works (opening the app 5 times before this message is shown) I'm not sure it will be enabled back soon

Reply:
This is an area that is not that easy to be automated and i agree i dont think it will be enabled soon. I will remove it.